charts/artifact-hub/values.yaml

```diff
@@ -119,12 +119,39 @@ hub:
     image:
       # Hub image repository (without the tag)
       repository: artifacthub/hub
+    # If you do want to specify resources, uncomment the following
```

Comment:
Thanks for adding those comments! Do you think it'd be clearer for users if we add them *below* the resources field? Just uncomment, remove braces and ready 😉

Reply:
I refactored/moved the comments.